edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/EdgeHubDesiredProperties.cs

```diff
@@ -1,19 +1,30 @@
-// Copyright (c) Microsoft. All rights reserved.
-namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
-{
-    using System;
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System;
     using System.Collections.Generic;
     using Microsoft.Azure.Devices.Edge.Util;
-    using Newtonsoft.Json;
-
-    public class EdgeHubDesiredProperties
-    {
+    using Microsoft.Azure.Devices.Edge.Util.Json;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// DTO that is used to deserialize EdgeHub Desired properties of the twin
+    /// into <see cref="EdgeHubConfig" /> by <see cref="TwinConfigSource" />.
+    /// </summary>
+    public class EdgeHubDesiredProperties
+    {
         [JsonConstructor]
-        public EdgeHubDesiredProperties(string schemaVersion, IDictionary<string, RouteConfiguration> routes, StoreAndForwardConfiguration storeAndForwardConfiguration)
+        public EdgeHubDesiredProperties(

```

Comment:
>EdgeHubDesiredProperties [](start = 15, length = 24)

chatted offline about this, we're moving towards a model where ver 1.0/1.1/1.2 will have separate JSON classes for deserialization, so that each individual version only parse the properties that it knows about.  This is okay for now, but we'll need to make sure to come back and split out the newly added properties.